db/migrate/20230213180133_change_webhooks_organization_id_type.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+class ChangeWebhooksOrganizationIdType < ActiveRecord::Migration[7.0]
+  def change
+    remove_column :webhooks, :organization_id
```

Comment:
Rails/ReversibleMigration: remove_column(without type) is not reversible. (https://rails.rubystyle.guide#reversible-migration, https://api.rubyonrails.org/classes/ActiveRecord/Migration/CommandRecorder.html)